### ISSUE TITLE
Head animations

### ISSUE THREAD

**Issue by MaxaAI** [Winfredy/SadTalker#749]
I just have a question because im using sadtalker in google colab is there an option i can make the head movements better that they don't look like watching around? i would like to have head movement but that its still straight

**Comment by Umargit1** [comment]
yes by increasing expression value in inference.py file from 1 to 2

**Comment by MaxaAI** [comment]
But i use google colab is there any opportunity?

**Comment by MaxaAI** [comment]
another question now the head is animated of the picture of myself that is nice. But it looks like it is cutting my neck and then animate everything. So the neck moves but that shouldn't be like that. How to fix that(in google colab!)

**Comment by liyuli1997** [comment]
> 另一个问题是，现在我的头像动画效果很好。但它看起来像是在割断我的脖子，然后把所有的东西都动画化。所以脖子会动，但不应该是这样。如何委员会（在google colab 中

您好，您的问题解决了吗，我也遇到了相同的问题
